frontend/README.md

```diff
@@ -0,0 +1,16 @@
+# HTML/CSS - Interview
+
+We believe that a good front-end should be built using semantic markup and well-structured CSS. To that end, it's important that our front-end developers demonstrate the ability to convert a mocked-up page image into well-written code. The goal of this excercise is to take the mockups provided and build an HTML/CSS representation of them.
```

Comment:
*nitpick* "front end" is the [correct noun](http://english.stackexchange.com/questions/34447/is-it-front-end-frontend-or-front-end).